libenv/sysinfo.c

```diff
@@ -1588,6 +1660,27 @@ static void OSClasses(EvalContext *ctx)
     else
     {
         SetFlavor2(ctx, "windows", major);
+        
+        // minor version        
+        char *minor = NULL;
+        int idx = (major - release) + strlen(major) + 1;
+        char *sp = &release[idx];
```

Comment:
I think this would yield the same result with a little less computing.
```suggestion
        char *sp = major + strlen(major) + 1;
```